source/github/github.go

```diff
@@ -20,6 +20,13 @@ import (
 const (
 	defaultURL = "https://github.com"     // Default GitHub URL
 	defaultAPI = "https://api.github.com" // Default GitHub API URL
+
+	// events for repo webhooks
+	eventPush         = "push"
```

Comment:
Any reason why we are duplicating these events here? These variables are already declared in `go-vela/types/constants` 👍 

Specifically, you can find them here:

https://github.com/go-vela/types/blob/80de769c682b6b87d09b6a76935cacea8ca8d78f/constants/event.go#L8-L23

Reply:
yes, actually. "comment" is the new event type we are using for repo and secret that we are rolling the multiple comment-related event types into (issue_comment, pull_request_review_comment, pull_request_review). we set it up this way to be easier to add the other pr related events to the "comment" event when we get to those stories (public project 0.4).

Reply:
reminder: each place in a pr to add a comment will trigger a different event. from the user's perspective, they likely don't know/care that they are different events. this also helps minimize the number of events a repo and secret will need to have enabled for the same functionality.

Reply:
👍 I see the difference now!